tests/solvers/quadratic/fgw_test.py

```diff
@@ -60,7 +59,12 @@ def test_gradient_marginals_fgw_solver(self, jit: bool):
 
     def reg_gw(a: jnp.ndarray, b: jnp.ndarray, implicit: bool):
       prob = quadratic_problem.QuadraticProblem(
-          geom_x, geom_y, geom_xy, fused_penalty=self.fused_penalty, a=a, b=b
```

Comment:
Not sure why it was reformatted, but would prefer to undo.

Reply:
done!